generators/turnip/templates/turnip.rake

```diff
@@ -0,0 +1,82 @@
+require 'rspec/core/rake_task'
+
+unless ARGV.any? { |a| a =~ /^gems/ } # Don't load anything when running the gems:* tasks
+  begin
+
+    namespace :turnip do
+      RSpec::Core::RakeTask.new(:ok, 'Run features that should pass') do |t|
+        t.pattern = './spec{,/*/**}/*.feature'
+        t.rspec_opts = '--tag default'
+      end
+
+      RSpec::Core::RakeTask.new(:wip, 'Run features that are being worked on') do |t|
+        t.pattern = './spec{,/*/**}/*.feature'
+        t.rspec_opts = '--tag wip'
+      end
+
+      RSpec::Core::RakeTask.new(:bug, 'Run features with known bugs') do |t|
+        t.pattern = './spec{,/*/**}/*.feature'
+        t.rspec_opts = '--tag bug'
+      end
+
+      RSpec::Core::RakeTask.new(:demo, 'Run demo feature') do |t|
```

Comment:
I guess we have to remove that for all BDD frameworks. That was introduced historically for CI and is no more required